guides/common/modules/proc_importing-an-ansible-playbook-by-name.adoc

```diff
@@ -1,21 +1,22 @@
+include::attributes-base.adoc[]
 
 [id="importing-an-ansible-playbook-by-name_{context}"]
 = Importing an Ansible Playbook by Name
 
-You can import an Ansible playbook from collections installed on the {SmartProxy}  by name.
+You can import Ansible playbooks by name to {Project} from collections installed on {SmartProxy}.
 
 .Procedure
-. On the target host, fetch the available Ansible playbooks.
+. Fetch the available Ansible playbooks using the following API request:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-curl -X GET 'Content-Type: application/json' https://_{smartproxy-example-com}_:{smartproxy_port}/ansible/api/v2/ansible_playbooks/fetch?proxy_id=_proxy_id_number_ | json_pp
+curl -X GET 'Content-Type: application/json' https://_{foreman-example-com}_:{smartproxy_port}/ansible/api/v2/ansible_playbooks/fetch?proxy_id=_proxy_id_number_ | json_pp
 ----
 . Select the Ansible playbook you want to import and note its name.
 . Import the Ansible playbook using its name:
 +
 [options="nowrap", subs="+quotes,verbatim,attributes"]
 ----
-curl -X PUT 'Content-Type: application/json' -d '{ "playbook_names": ["_playbook_name_"] }' https://_{smartproxy-example-com}_:{smartproxy_port}/ansible/api/v2/ansible_playbooks/sync?proxy_id=_proxy_id_number_ | json_pp
+curl -X PUT 'Content-Type: application/json' -d '{ "playbook_names": ["_playbook_name_"] }' https://_{foreman-example-com}_:{smartproxy_port}/ansible/api/v2/ansible_playbooks/sync?proxy_id=_proxy_id_number_ | json_pp
```

Comment:
```suggestion
curl -X PUT 'Content-Type: application/json' -d '{ "playbook_names": ["_My_Playbook_Name_"] }' https://_{foreman-example-com}_:{smartproxy_port}/ansible/api/v2/ansible_playbooks/sync?proxy_id=_My_smart-proxy-context_ID_ | json_pp
```

With Hammer CLI, we mostly prefix `My_` to all values. I think this makes sense here as well.